sdks/python/hatchet_sdk/worker/runner/runner.py

```diff
@@ -121,6 +230,11 @@ def __init__(
         self.lifespan_context = lifespan_context
         self.log_sender = log_sender
 
+        self._batch_controllers: dict[str, _BatchController] = {}
+        for action_id, task in self.action_registry.items():
+            if getattr(task, "is_batch", False):
+                self._batch_controllers[action_id] = _BatchController(action_id, task)
```

Comment:
am I understanding this wrong? this loops through the action registry and if something is _not_ a batch task, creates a batch controller for it?